server/db.mjs

```diff
@@ -2,6 +2,12 @@ import dotenv from "dotenv";
 import pgp from "pg-promise";
 
 const db = initDb();
+
+// do some joining because Good Girl
+export const getJoined = async () =>
+  await db.any(
+    "SELECT DISTINCT ON (specimens.name) specimens.name AS nickname, specimens.url AS url, species.name AS species, sightings.location AS recent_sightings FROM specimens INNER JOIN species ON specimens.species_id = species.species_id INNER JOIN sightings ON sightings.specimen_id = specimens.specimen_id",
```

Comment:
On really long lines like this I find it personally easier to read if there is some wrapping done, jus prevents people from having to scroll

Reply:
Good job with your joins